### ISSUE TITLE
Vue2 support

### ISSUE THREAD

**Issue by titusdecali** [mujahidfa/vue-boring-avatars#1]
I know you've written this in the repo already, but I'd like to chime in with a great need for Vue 2 support.

**Comment by mujahidfa** [comment]
@titusdecali thanks for the suggestion. Vue 2 support is definitely in the works, but I'm currently still figuring out what is the best way to go about this. Either to:
1. Build a separate package for Vue 2, which is relatively simple to achieve, however will require a lot of duplication between Vue 2 and Vue 3 code.
2. Use something like [vue-demi](https://github.com/vueuse/vue-demi) to support both Vue 2 & 3 out of the box in one single package. However, it might result in a bigger package size since it requires Vue 2 users to install the Composition API, which might not be desirable.

Any thoughts on this?

**Comment by titusdecali** [comment]
I think the best route would be to make a generic js package that we can import into any type of project, whether it be React, Vue 2/3, or otherwise.

Since I posted the above, I've found that using the source link from the original BoringAvatars repo 
(ex: https://source.boringavatars.com/marble/40) and just using string literals within a custom component works really well on its own. I think I'll continue to do things this way which will ensure that I can use any updates they add to the main package.

Example:

![image](https://user-images.githubusercontent.com/4532534/120909172-55705880-c6ad-11eb-9a8e-e51ce2cacb14.png)
